src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,64 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
```

Comment:
The `getAll` method for `searchParams` returns an array, so the fallback to an empty array `[]` is unnecessary. You can safely remove `|| []` as `getAll` will always return an array.